### PR TITLE
Fix displaying prepared query in UI

### PR DESCRIPTION
**Pull request by wendigo**
Fixes https://github.com/prestosql/presto/issues/2784

Screenshots:

Zrzut ekranu 2020-02-18 o 13 43 05
Zrzut ekranu 2020-02-18 o 13 43 22

### CODE REVIEW

**Comment by wendigo** [comment]
@sopel39 it is hidden for non prepared statements. Also I've added syntax higlighting to prepared query as well (looks more consistent)

**Comment by sopel39** [comment]
please rebase

**Comment by sopel39** [comment]
please rebase

File: presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx [hunk 1]
```diff
@@ -982,7 +986,7 @@ export class QueryDetail extends React.Component {
 
     renderPreparedQuery() {
         const query = this.state.query;
-        if (query.preparedQuery === null) {
+        if (!query.hasOwnProperty('preparedQuery') || query.preparedQuery === null) {
```
Comment (findepi) [inline presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx]: why not 

```
if (query.preparedQuery === undefined || query.preparedQuery === null)
```

?

Comment (findepi) [reply presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx]: or 
```
if (!query.preparedQuery)
```

Comment (wendigo) [reply presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx]: Habit + query has no shape defined in React State

**Comment by wendigo** [comment]
@findepi done

**Pull request merged**